### PR TITLE
launch admin should be required

### PR DESCRIPTION
**Pull request by wantedsystem**
Result =>
Capture d’écran 2021-11-01 à 16 38 02

### CODE REVIEW

File: src/newLaunch/lbp/stages/stage4.ts [hunk 1]
```diff
@@ -291,6 +291,8 @@ export class Stage4 extends BaseStage<ILbpConfig> {
     } else if (!(Number.parseInt(endTimes[1]) >= 0)
       || !(Number.parseInt(endTimes[1]) < 60)) {
       message = "Please enter a valid value for End Time";
+    } else if (!Utils.isAddress(this.launchConfig.launchDetails.adminAddress)) {
+      message = "Please enter a valid value for launch administrator";
```
Comment (dkent600) [inline src/newLaunch/lbp/stages/stage4.ts:295]: Lets change the text to : "Please enter a valid address for the LBP administrator"

**Review by dkent600 — CHANGES_REQUESTED** [review]
Do we need to fix this for the seed wizard too?

**Comment by wantedsystem** [comment]
> Do we need to fix this for the seed wizard too?

Already exists

File: src/newLaunch/lbp/stages/stage4.ts (hunk 1 above)
Comment (wantedsystem) [reply src/newLaunch/lbp/stages/stage4.ts]: I changed to : "Please enter a valid wallet address for LBP Administrator", like the seed wizard

Comment (dkent600) [reply src/newLaunch/lbp/stages/stage4.ts]: ok, but for correct english, it should be "the LBP Administrator", in both places ("the Seed Administrator" for seeds).

Comment (wantedsystem) [reply src/newLaunch/lbp/stages/stage4.ts]: Done

**Review by dkent600 — APPROVED** [review]

**Pull request merged**